envpool/python/data.py

```diff
@@ -124,8 +125,7 @@ def gymnasium_spec_transform(
   )
 
 
-def dm_structure(root_name: str,
-                 keys: List[str]) -> List[Tuple[List[str], int]]:
+def dm_structure(root_name: str, keys: List[str]) -> PyTreeSpec:
```

Comment:
```suggestion
def dm_structure(
  root_name: str,
  keys: List[str],
) -> Tuple[List[Tuple[int, ...]], List[int], PyTreeSpec]:
```

NOTE: Here the `treespec` represents a nested `namedtuple`. The path to a leaf is a `tuple` of `int`s rather than a `tuple` of `str`s. In `gym_structure`, the returned `treespec` represents a nested `dict` and the path is a `tuple` of `str`s.